README.md

```diff
@@ -185,9 +198,29 @@ style guide.
   e = M * c**2
   ```
 
+* <a name="spaces-braces"></a>
+  No spaces after `(`, `[` or before `]`, `)`.
+  Use spaces around `{` and before `}`.
+<sup>[[link](#spaces-braces)]</sup>
+
+  ```Ruby
+  # bad
+  some( arg ).other
+  [ 1, 2, 3 ].each{|e| puts e}
+
+  # good
+  some(arg).other
+  [1, 2, 3].each { |e| puts e }
+  ```
+
   `{` and `}` deserve a bit of clarification, since they are used
   for block and hash literals, as well as string interpolation.
+
   For hash literals two styles are considered acceptable.
+  The first variant is slightly more readable (and arguably more
```

Comment:
I don't think this conflicted, so this may be a new place that we need to remove that again.